### DISCUSSION TITLE
[TOOL] PvPstats: the missing PvP statistics system for WoW

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3907] [Tools]
PvPstats is a tool that I've created some years ago as part of my [Master's degree thesis about WoW emulation](https://community.trinitycore.org/topic/13025-just-thank-you-wow-emulation/). It allows to **store and display data about Battleground scores.**

Basically the PvPstats system is composed by a core modification (already included in AC) + a web application.

You can find more details & setup instructions here:

https://github.com/FrancescoBorzi/PvPstats

------------

- There is also a [list](http://shinworld.altervista.org/PvPstats/) of servers that use/used the PvPstats

- Some screenshots:

![PvPstats Index](http://shinworld.altervista.org/PvPstats/img/index.png "PvPstats Index")
![PvPstats BattleGround search](http://shinworld.altervista.org/PvPstats/img/battleground1.png "PvPstats BattleGround search")
![PvPstats BattleGround score](http://shinworld.altervista.org/PvPstats/img/battleground2.png "PvPstats BattleGround score")
![PvPstats Top100](http://shinworld.altervista.org/PvPstats/img/top100.png "PvPstats Top100")

**Comment by Naxxramas-Server** [comment]
Very nice ! Do u know aproximately same tool like yours about PVE ?

**Comment by Helias** [comment]
@Naxxramas-Server  https://github.com/Helias/TC-PvEstats

Demo: http://usefulness.altervista.org/PvEstats/
